libraries/package-extractor/src/tests/package-extractor-test-repo/packages/foo/node_modules/.pnpm/baz/package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
The node_modules directory should not be checked in and should be gitignored. If this is required, an install should be performed as a part of the build process.

Reply:
Ahh hmm, this is intentional. Seems a bit janky of a way to test this... it's not actually an install, so it doesn't have symlinks etc. that would normally exist... Yeah, I would suggest doing an actual install to get the state on-disk to match what you plan to test. Really, you could make two test projects within the repo, and one of them would run the deploy and validate output content. You can add the deploy scenario to the config in the Rushstack repo, so that everything should be prepared for you.

Reply:
Yes, just created a fake node_modules structure for testing. It had symlinks like pnpm created but without hardlinks to pnpm global store.

1. packages/foo/node_modules/bar - > packages/bar                                                 (local project dependencies)
2. packages/foo/node_modules/baz -> packages/foo/node_modules/.pnpm/baz    (third party dependencies)

Reply:
Two test projects inside the repo only mocks the local project dependency scenario...